data/remnant missions.txt

```diff
@@ -495,3 +560,718 @@ mission "Remnant: Technology Available"
+	on stopover
+		conversation
+			`Once again you have successfully dodged the Archon to land on Nasqueron. Somehow, knowing it was going to be there was both better and worse than the surprise of the first time.`
+			`	As <ship> sinks through the cloud layers, you gently activate the repulsors to slow your descent to avoid startling the void sprites. After what seems like an eternity, you notice a few nearby sprites descending as well, and you match velocity with them, slowing as you reach the altitude listed in the log.`
+			`	Setting the ship on auto-pilot, you head back and suit-up in something that looks more reminiscent of the old deep-sea diving suits than anything carried on a modern ship. Readying the eggs in the cargo hold, you equalize pressure in the main cargo hold to the swirling clouds outside, and open the bay doors. As predicted, at this pressure level, the eggs are almost weightless, and are fairly easy to carry to the door of the ship and release.`	
+			`	By the time you finish releasing the eggs back into the clouds, several void sprites have converged on your ship and appear to be herding the eggs away from you with gentle buffets of their wings. You take a moment to appreciate the alien beauty of these creatures before closing the hatch and purging the ship with fresh air.`
+	on complete
+		"reputation:  Drak " = 1
+		conversation
+			`Your return trip to Aventine is just as exciting, dodging the Archon on the way out of the system.`
+			`	Back on the ground, you report on how the release went. The researcher is pleased to hear that the void sprites responded to their presence. "We will give them a few weeks, then try sending another ship to monitor them. Hopefully the Archon will be pacified by this. Meet me in the alien environments lab just off the spaceport if you are interested in pursuing this."`
+
+
+
+mission "Remnant: Return the Samples 2"
+	name "Return the Samples"
+	description "A researcher on <planet> has asked you to return some more samples to Slylandro."
+	source Aventine
+	stopover Slylandro
+	to offer
+		has "Remnant: Return the Samples: done"
+	on offer
+		conversation
+			`Following Plume's directions, you find a lab tucked behind the main area of the spaceport, where a short walkway leads into an underground structure. Stepping inside the door you note a rack of camouflage netting positioned easily at hand. Given what you have learned of their history, it is probably a last ditch preparation in case they need to hide the entrance from a raid.`
+			`	The lab itself is a microcosm of Remnant development, the structure appears to be some strange semi-organic material, and you can see an eclectic mix of familiar and unfamiliar shapes cluttering the tables. If you had to guess, you would think that this lab was the result of several different families of technology, only one of which was human.`
```

Comment:
I'd drop this paragraph.

Reply:
I would prefer to leave this paragraph in for the exact reason that jafdy gives.

Reply:
It doesn’t tell us anything we don’t already know; yeah, the Remnant include a lot of stolen and/or found alien tech in their work. We get it.

Reply:
ES is not an exercise in minimalism. Some repetition is useful if it sets up the atmosphere properly.

And that being said, yes, *you* get it, so will most of the others who have played every mission a dozen times. What about the player who is meeting the Remnant for the first time? They haven't internalized the description of every ship, outfit, and early mission yet. A description like that helps bring them into the setting more. Such a player may well only have a vague recollection that someone said something about how they lure the Korath in so they can salvage their outfits, and maybe something about having found alien ruins when they first came out here? That's really not enough for someone to go on when trying to understand a culture (and their character obviously would know far more about it, being actually exposed to it in ways the player is not)